packages/base/src/keybindings.json

```diff
@@ -33,5 +53,175 @@
     "command": "jupytercad:selectConsoleCompleter",
     "keys": ["Enter"],
     "selector": ".jpcad-console .jp-ConsolePanel .jp-mod-completer-active"
+  },
+  {
+    "command": "jupytercad:sketch",
+    "keys": ["Accel Alt N"],
+    "selector": ".jcad-Mainview"
+  },
+  {
+    "command": "jupytercad:sketch",
+    "keys": ["Accel Alt N"],
+    "selector": ".jpcad-sidepanel-widget"
+  },
+  {
+    "command": "jupytercad:newBox",
+    "keys": ["Accel Alt B"],
+    "selector": ".jcad-Mainview"
+  },
+  {
+    "command": "jupytercad:newBox",
+    "keys": ["Accel Alt B"],
+    "selector": ".jpcad-sidepanel-widget"
+  },
+  {
+    "command": "jupytercad:newCylinder",
+    "keys": ["Accel Alt C"],
+    "selector": ".jcad-Mainview"
+  },
+  {
+    "command": "jupytercad:newCylinder",
+    "keys": ["Accel Alt C"],
+    "selector": ".jpcad-sidepanel-widget"
+  },
```

Comment:
I wonder if we should introduce a data attribute in our components that should receive keybinding events, like `data-jcad-keybinding` or something. Then we could get rid of this duplication and just do something like:

```
  {
    "command": "jupytercad:newCylinder",
    "keys": ["Accel Alt C"],
    "selector": "[data-jcad-keybinding]"
  }
```

Reply:
I can try on it, thanks for suggesting.

Reply:
This worked, thanks @martinRenou.
Done in [`2ddc2ed`](https://github.com/jupytercad/JupyterCAD/pull/451/commits/2ddc2ed3f36e63b4338206226fdbb9934bff2a30)

Reply:
Ah I meant using a data attribute, not a class, have you tried that? But a general purpose class like you did sounds good too.

Reply:
Oh sorry, I guess I misinterpreted the idea. I'd be happy to explore this option too if it's more viable.
Might be silly but I'd just want to confirm how do I add data attribute to the component?